### PR TITLE
Update ui-class.md new project instructions to VS 2019

### LINKED ISSUE #14946: No Razor Class Library in Visual Studio
I chose "ASP.NET Core Web Application" as project type, named the project but in the list there is no Class Library (ASP.NET Core 3.0 is selected):

There is also a project type named "Razor Class Library" but it creates a Blazor library. I am having latest VS 16.4 Preview 1.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9fd019c7-ceb7-6ef3-26ba-6b44341b6fc5
* Version Independent ID: fd116d00-0dc7-efaf-c714-34d77b345a05
* Content: [Reusable Razor UI in class libraries with ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/razor-pages/ui-class?view=aspnetcore-3.0&tabs=visual-studio)
* Content Source: [aspnetcore/razor-pages/ui-class.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/razor-pages/ui-class.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-razorpages**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by wadepickett**
Fixes #14946 

[Internal Review](https://review.docs.microsoft.com/en-us/aspnet/core/razor-pages/ui-class?view=aspnetcore-2.1&branch=pr-en-us-15310&tabs=visual-studio)

Updated VS instructions for new project to VS 2019.  

(Tested CLI new project instructions, they are good as is.)

### CODE REVIEW

File: aspnetcore/razor-pages/ui-class.md [hunk 1]
```diff
@@ -22,13 +22,13 @@ Razor views, pages, controllers, page models, [Razor components](xref:blazor/cla
 
 # [Visual Studio](#tab/visual-studio)
 
-* From the Visual Studio **File** menu, select **New** > **Project**.
-* Select **ASP.NET Core Web Application**.
-* Name the library (for example, "RazorClassLib") > **OK**. To avoid a file name collision with the generated view library, ensure the library name doesn't end in `.Views`.
-* Verify **ASP.NET Core 3.0** or later is selected.
-* Select **Razor Class Library** > **OK**.
+* From the Visual Studio **File** menu, select **New**, then select **Project**.
```
Comment (Rick-Anderson) [inline aspnetcore/razor-pages/ui-class.md]: Create a new project.

File: aspnetcore/razor-pages/ui-class.md [hunk 2]
```diff
@@ -22,13 +22,13 @@ Razor views, pages, controllers, page models, [Razor components](xref:blazor/cla
...
+* Select **Razor Class Library**, then select **Next**.
```
Comment (Rick-Anderson) [inline aspnetcore/razor-pages/ui-class.md]: Go back to the original and use `>` in lieu of  `,then select`. Do this everywhere you made that change. > is much preferred.

File: aspnetcore/razor-pages/ui-class.md [hunk 3]
```diff
@@ -22,13 +22,13 @@ Razor views, pages, controllers, page models, [Razor components](xref:blazor/cla
...
+* Name the library (for example, "RazorClassLib"), then select **Create**. To avoid a file name collision with the generated view library, ensure the library name doesn't end in `.Views`.
+* Verify **ASP.NET Core 3.0** or later is selected. The **Razor Class Library** should remain selected.
```
Comment (Rick-Anderson) [inline aspnetcore/razor-pages/ui-class.md]: Delete this entire line.  2.1 and higher is supported. This is not a 3.0 version - right?

File: aspnetcore/razor-pages/ui-class.md [hunk 4]
```diff
@@ -22,13 +22,13 @@ Razor views, pages, controllers, page models, [Razor components](xref:blazor/cla
...
+* Select **Support pages and views**, then select **Create**.
```
Comment (Rick-Anderson) [inline aspnetcore/razor-pages/ui-class.md]: revert

Comment (Rick-Anderson) [reply aspnetcore/razor-pages/ui-class.md]: That's only an option in 3.0

You could add. 

For ASP.NET 3.0, select **Support pages and views** if you need to support views. By default, only Razor Pages are supported.

File: aspnetcore/razor-pages/ui-class.md [hunk 5]
```diff
@@ -22,13 +22,13 @@ Razor views, pages, controllers, page models, [Razor components](xref:blazor/cla
...
 
-The Razor class library (RCL) template defaults to Razor component development by default. A template option in Visual Studio provides template support for pages and views.
+The Razor class library (RCL) template defaults to Razor component development by default. The **Support pages and views** template option in Visual Studio provides template support for pages and views.
```
Comment (Rick-Anderson) [inline aspnetcore/razor-pages/ui-class.md]: The Razor class library (RCL) template defaults to Razor component development by default. The **Support pages and views** option supports pages and views.

File: aspnetcore/razor-pages/ui-class.md (hunk 3 above)
Comment (wadepickett) [reply aspnetcore/razor-pages/ui-class.md]: The moniker range in the original doc specified >= 3.0 for this section.  So I thought this section was specific to 3.0.  Will remove line.

File: aspnetcore/razor-pages/ui-class.md (hunk 2 above)
Comment (wadepickett) [reply aspnetcore/razor-pages/ui-class.md]: Thanks. I am leaving "select" in this sentence.  It is a bit confusing having > at the end of a lot text that is not the step itself.   "* For ASP.NET 3.0, select **Support pages and views** if you need to support views. By default, only Razor Pages are supported. Select **Create**."

File: aspnetcore/razor-pages/ui-class.md (hunk 3 above)
Comment (Rick-Anderson) [reply aspnetcore/razor-pages/ui-class.md]: LIne 5 has `monikerRange: '>= aspnetcore-2.1'` so I assumed this was not using monikers. We don't mix metadata monikers with inline monikers. If this article has multiple version, then keep the version specific code. I didn't see that in the review diffs.

File: aspnetcore/razor-pages/ui-class.md [hunk 6]
```diff
@@ -4,7 +4,7 @@ author: Rick-Anderson
 description: Explains how to create reusable Razor UI using partial views in a class library in ASP.NET Core.
 monikerRange: '>= aspnetcore-2.1'
```
Comment (Rick-Anderson) [inline aspnetcore/razor-pages/ui-class.md]: Remove this

File: aspnetcore/razor-pages/ui-class.md (hunk 4 above)
Comment (Rick-Anderson) [reply aspnetcore/razor-pages/ui-class.md]: Given this is 3..0 specific, keep the instructions specific to 3.0
Sorry, I was going off line 5

File: aspnetcore/razor-pages/ui-class.md (hunk 3 above)
Comment (wadepickett) [reply aspnetcore/razor-pages/ui-class.md]: Thanks, got it.  The topic original (published) had the mixed metadata + inline moniker problem.  I didn't notice the metadata moniker until you pointed it out. I will keep a watch out for that issue.

**Pull request merged**